source/index.txt

```diff
@@ -27,7 +27,7 @@ processing and reactive streams, we recommend the `Reactive Streams Driver
 
 The official MongoDB Java driver lets you connect your Java applications
 to MongoDB and work with your data. On this page, you can find descriptions of
-each section of the driver documentation.
+each section of the driver documentation and where to go for more resources.
```

Comment:
I think "resources" doesn't help the reader understand what it is. See the other PR review comment on the "Resources" section.